Error: Cannot find module 'espree'

When changing from `~2.10.0` (resolved as `2.10.0`) to `~2.11.0` (resolved as `2.11.1`)
![image](https://github.com/ota-meshi/eslint-plugin-jsonc/assets/2230835/3eae689f-812e-4e17-a7e1-1b1f6ec804f5)

I am getting the following error when running using `pnpm`:
```
Error: Cannot find module 'espree'
Require stack:
- /home/runner/node_modules/.store/eslint-plugin-jsonc-virtual-f2fb243113/node_modules/eslint-plugin-jsonc/dist/utils/eslint-ast-utils.js
- /home/runner/node_modules/.store/eslint-plugin-jsonc-virtual-f2fb243113/node_modules/eslint-plugin-jsonc/dist/rules/array-bracket-newline.js
- /home/runner/node_modules/.store/eslint-plugin-jsonc-virtual-f2fb243113/node_modules/eslint-plugin-jsonc/dist/utils/rules.js
- /home/runner/node_modules/.store/eslint-plugin-jsonc-virtual-f2fb243113/node_modules/eslint-plugin-jsonc/dist/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Module.require (node:internal/modules/cjs/loader:1235:19)
    at require (node:internal/modules/helpers:176:18)
    at Object.<anonymous> (/home/runner/node_modules/.store/eslint-plugin-jsonc-virtual-f2fb243113/node_modules/eslint-plugin-jsonc/dist/utils/eslint-ast-utils.js:5:18)
    at Module._compile (node:internal/modules/cjs/loader:1376:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
    at Module.load (node:internal/modules/cjs/loader:1207:32)
    at Module._load (node:internal/modules/cjs/loader:1023:12)
    at Module.require (node:internal/modules/cjs/loader:1235:19)
```

---

I think we need to add `espree` into the `dependencies`, or into the `peerDependencies`.

--- a/package.json
+++ b/package.json
@@ -68,6 +68,7 @@
   "dependencies": {
     "@eslint-community/eslint-utils": "^4.2.0",
     "eslint-compat-utils": "^0.1.2",
+    "espree": "^9.6.1",
     "graphemer": "^1.4.0",
     "jsonc-eslint-parser": "^2.0.4",
     "natural-compare": "^1.4.0"

